OMCompiler/Compiler/NBackEnd/Modules/2_Pre/NBAlias.mo

```diff
@@ -299,7 +314,7 @@ protected
     // 2. balance sets - choose variable to keep if necessary
     // 3. match/sort set (linear w.r.t. vars since all equations contain two crefs at max and are simple/linear)
     // --------------------------------------------------------------------------------------------------------
-    replacements := UnorderedMap.new<Expression>(ComponentRef.hash, ComponentRef.isEqual, size);
+    replacements := UnorderedMap.new<Expression>(ComponentRef.hash, ComponentRef.isEqual, size); // kab
```

Comment:
Please remove comments like `// kab`, i think i made them to show you where to find things.

Reply:
I removed it, you can see the change in the new commit.